docs/src/main/sphinx/connector/iceberg.md

```diff
@@ -681,13 +681,8 @@ TABLE </sql/show-create-table>`.
 
 Table properties supply or set metadata for the underlying tables. This is key
 for {doc}`/sql/create-table-as` statements. Table properties are passed to the
-connector using a {doc}`WITH </sql/create-table-as>` clause:
+connector using a {doc}`WITH </sql/create-table-as>` clause.
 
-```
-CREATE TABLE tablename
-WITH (format='CSV',
```

Comment:
This is definitely wrong for Iceberg...

Reply:
That's good removal.
We have anyway an example of concrete table properties after the _Iceberg table properties_ table.